zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -25,11 +25,15 @@
       <term name="note">wie Anm.</term>
       <term name="et-al">et al.</term>
       <term name="editor" form="short">Hg.</term>
+      <term name="translator" form="verb-short">übers. von</term>
+      <term name="editortranslator" form="verb-short">hg. und übers. von</term>
     </terms>
   </locale>
   <locale xml:lang="en">
     <terms>
       <term name="note">as note</term>
+      <term name="translator" form="verb-short">transl. by</term>
+      <term name="editortranslator" form="verb-short">ed. and transl. by</term>
```

Comment:
Okay, then you can delete the first line completely and adjust the second line accordingly.